CONTRIBUTING.md

```diff
@@ -1,122 +1,172 @@
-
-In this guide you will find:
+This guide will help you get started with the Mule Runtime's development
+environment. You will also learn the set of rules you are expected to
+follow in order to submit improvements and/or fixes to these projects.
 
 - [Before you begin](#before-you-begin) 
-    - [Getting to know better Mule](#getting-to-know-better-mule) 
-    - [Visiting the community meeting points](#visiting-the-community-meeting-points)
-    - [Understanding the extension mechanisms](#understanding-the-extension-mechanisms)
-- [Setting up the development environment](#setting-up-the-development-environment) 
+    - [Starting Points](#starting-points) 
+    - [Visiting the Community Meeting Points](#visiting-the-community-meeting-points)
+    - [Understanding the Extension Mechanisms](#understanding-the-extension-mechanisms)
+- [Setting Up the Development Environment](#setting-up-the-development-environment) 
     - [Installing Prerequisites](#installing-prerequisites) 
     - [Getting the Source Code](#getting-the-source-code) 
 - [Configuring the IDE](#configuring-the-ide)
     - [Working with Eclipse](#working-with-eclipse)
     - [Working with IntelliJ IDEA](#working-with-intellij-idea) 
-- [Developing your contribution](#developing-your-contribution)
-    - [Creating your topic branch](#creating-your-topic-branch)
-    - [Updating Your topic Branch](#updating-your-topic-branch)
+- [Developing Your Contribution](#developing-your-contribution)
+    - [Creating Your Feature Branch](#creating-your-feature-branch)
+    - [Updating Your Feature Branch](#updating-your-feature-branch)
     - [Submitting a Pull Request](#submitting-a-pull-request)
 - [Summary](#summary)
 
+
 -----------------------------------------------------------------------------
 
  
-# Before you begin 
-Mule is a powerful and complex project. Before contributing to the Mule source code, it's important to understand the domain of the enterprise integration, Mule Runtime from the user point of view and the different mechanisms to extend Mule.
+## Before You Begin 
+
+Mule is a powerful and complex project. Before contributing to the Mule source code, it is important to understand the domain of the Enterprise Integration, the Mule Runtime from the user point of view and the different mechanisms to extend Mule.
 
-## Getting to know better Mule
-There are a number of sources you can use to understand better Mule and its forthcoming features:
+### Starting Points
+
+There are many resources available to help you become familiar with Mule and its features:
 
 - MuleSoft's [blog](http://blogs.mulesoft.com/). The fastest way of knowing about new features in mule. 
-- [@MuleSoft](https://twitter.com/MuleSoft)'s twitter account. You might also want to check the [#MuleESB](https://twitter.com/hashtag/MuleESB) hashtag.
 - [Mule User Guide](https://developer.mulesoft.com/docs/display/current/Mule+User+Guide), the official documentation around usage of Mule Runtime.
```

Comment:
Should point to https://docs.mulesoft.com/mule-user-guide which will display the latest version.